### PR TITLE
Makes last name optional

### PR DESCRIPTION
**Pull request by bsonntag**
Fixes issue https://github.com/groupbuddies/council/issues/119

HQ does not garantees the existance of a last name.
For this reason I made the last name an optional field in Council.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -9,13 +9,23 @@ class User < ActiveRecord::Base
...
   end
+
+  private
+
+  def full_name
+    if last_name.present?
+      [first_name, last_name].join(' ')
+    else
+      first_name
+    end
```
Comment (naps62) [inline app/models/user.rb]: this whole statement can be refactored to:

```ruby
[first_name, last_name].join(' ').strip
```

File: db/migrate/20150408155101_changes_users_optional_fields.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangesUsersOptionalFields < ActiveRecord::Migration
+  def change
+    change_column_null :users, :last_name, false
```
Comment (naps62) [inline db/migrate/20150408155101_changes_users_optional_fields.rb]: shouldn't this be `true`?

File: app/models/user.rb (hunk 1 above)
Comment (bsonntag) [inline app/models/user.rb]: Good point. Done.

File: db/migrate/20150408155101_changes_users_optional_fields.rb (hunk 2 above)
Comment (bsonntag) [inline db/migrate/20150408155101_changes_users_optional_fields.rb]: Sorry about that. Fixed.

**Comment by bsonntag** [comment]
Should I merge?

**Comment by naps62** [comment]
@bsonntag :+1: but you'll have to use the button here, since this comes from your fork

**Pull request merged**